CANCode/CANCode.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
What's the purpose of the empty file?

Reply:
Made it while making directory, just deleted it